### PR TITLE
Lower the attached gas to fit 2746

### LINKED ISSUE #109: init-near-contracts fail at call init on eth-client contract
It's reproducible to me everytime, but seems okay to @nearmax ,  Maybe a nodejs version specific issue. My reproduce:
```
node index.js clean
node index.js prepare
node index.js start near-node
node index.js start ganache
./scripts/start_ganache.sh # in a separate window
node index.js init-near-contracts

Key to call Eth2NearClient contract is not specified. Reusing master key.
Key to call Eth2NearProver contract is not specified. Reusing master key.
Creating accounts and deploying the contracts.
Account eth2nearclient does not exist creating it.
Created account eth2nearclient
Deployed contract to account eth2nearclient
Account eth2nearprover does not exist creating it.
Created account eth2nearprover
Deployed contract to account eth2nearprover
Initializing client and prover contracts.
EthClient is not initialized, initializing...
TxHash 7J8LwQSRQmtUWUJBGpeFpLjYW1RvAu6EXMxjQKqHot67
Failed:  InvalidTxError: {}
    at Object.parseRpcError (/Users/bo/workspace/rainbow-bridge/environment/node_modules/nearlib/lib/utils/rpc_errors.js:25:19)
    at JsonRpcProvider.sendJsonRpc (/Users/bo/workspace/rainbow-bridge/environment/node_modules/nearlib/lib/providers/json-rpc-provider.js:60:40)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async signAndSendTransaction (/Users/bo/workspace/rainbow-bridge/environment/lib/borsh/index.js:203:20)
    at async Eth2NearClientContract.value [as init] (/Users/bo/workspace/rainbow-bridge/environment/lib/borsh/index.js:246:43)
    at async Eth2NearClientContract.maybeInitialize (/Users/bo/workspace/rainbow-bridge/environment/lib/eth2near-client-contract/index.js:114:13)
    at async Command.execute (/Users/bo/workspace/rainbow-bridge/environment/commands/init-near-contracts.js:64:9)
    at async Promise.all (index 0)
    at async /Users/bo/workspace/rainbow-bridge/environment/index.js:365:16 {
  type: 'InvalidTxError'
}
(node:33437) UnhandledPromiseRejectionWarning: Error: {}
    at Object.parseRpcError (/Users/bo/workspace/rainbow-bridge/environment/node_modules/nearlib/lib/utils/rpc_errors.js:25:19)
    at JsonRpcProvider.sendJsonRpc (/Users/bo/workspace/rainbow-bridge/environment/node_modules/nearlib/lib/providers/json-rpc-provider.js:60:40)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async signAndSendTransaction (/Users/bo/workspace/rainbow-bridge/environment/lib/borsh/index.js:203:20)
    at async Eth2NearClientContract.value [as init] (/Users/bo/workspace/rainbow-bridge/environment/lib/borsh/index.js:246:43)
    at async Eth2NearClientContract.maybeInitialize (/Users/bo/workspace/rainbow-bridge/environment/lib/eth2near-client-contract/index.js:114:13)
    at async Command.execute (/Users/bo/workspace/rainbow-bridge/environment/commands/init-near-contracts.js:64:9)
    at async Promise.all (index 0)
    at async /Users/bo/workspace/rainbow-bridge/environment/index.js:365:16
(node:33437) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:33437) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

### PR DESCRIPTION
**Pull request by nearmax**
Fixes #109 . The code was broken by https://github.com/nearprotocol/nearcore/pull/2746

### CODE REVIEW

File: libs-rs/eth-client/Cargo.toml [hunk 1]
```diff
@@ -9,14 +9,15 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 eth-types =  { path = "../eth-types" }
-near-sdk = "0.11.0"
-borsh = "0.6.2"
+near-bindgen = "0.4.1"
+borsh = "0.2.10"
```
Comment (ailisp) [inline libs-rs/eth-client/Cargo.toml:13]: I think this is a mistake? These dependencies are old

File: libs-rs/eth-client/build.sh [hunk 2]
```diff
@@ -3,7 +3,5 @@
 # Exit script as soon as a command fails.
 set -e
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
-RUSTFLAGS='-C link-arg=-s' cargo +stable build --target wasm32-unknown-unknown --release
-cp $DIR/../target/wasm32-unknown-unknown/release/eth_client.wasm $DIR/../res/
+RUSTFLAGS='-C link-arg=-s' cargo build --target wasm32-unknown-unknown --release
```
Comment (ailisp) [inline libs-rs/eth-client/build.sh:6]: Also this, and below usage of near_bindgens

**Comment by ailisp** [comment]
Does only change amount of gas fix?

**Comment by ailisp** [comment]
I see the revert is deliberate, is it possible to use a one month ago nearcore + borsh + near-sdk? Otherwise eth-prover test would fail (VMRuntimeStandalone need recent borsh/near-sdk to compile, #25 )

**Comment by nearmax** [comment]
Closing in favor of https://github.com/near/rainbow-bridge/pull/117

**Comment by nearmax** [comment]
> I see the revert is intentional, is it possible to use a one month ago nearcore + borsh + near-sdk? Otherwise eth-prover test would fail (VMRuntimeStandalone need recent borsh/near-sdk to compile, #25 )

I accidentally pushed the revert.

**Pull request closed without merging**